runtime/common/src/lib.rs

```diff
@@ -240,6 +240,11 @@ pub mod parachains {
 		pub const ID: u32 = 2000;
 		pub const KUSD_KEY: &[u8] = &[0, 129];
 	}
+
+	pub mod altair {
+		pub const ID: u32 = 2088;
+		pub const AIR_KEY: &[u8] = &[0, 1];
```

Comment:
What is the purpose of the AIR_KEY? It uniquely identifies a chain within the local knowledge base?

Reply:
Good question! The idea is that chains and tokens IDs in XCM are canonical, so having those for our own chain as well helps with clarity by having it also listed alongside other chains we integrate with.

The AIR_KEY value itself is arbitrary but it must be immutable as it's used as the GeneralKey value when we represent the token in XCM MultiLocation. In development, we were creating that value by doing CurrencyId::Native::encode() but that may change silently if we change the order of variants in that enum type.